stable/fairwinds-insights/values.yaml

```diff
@@ -351,6 +351,26 @@ databaseCleanupCronjob:
     # -- The user ID to run the database cleanup job under.
     runAsUser: 10324
 
+resourcesRecommendationsCronjob:
+  # -- Enable resources recommendations true by default
+  enabled: true
+  # -- Resources for the resources recommendations job.
+  resources:
+    limits:
+      cpu: 500m
+      memory: 1024Mi
+    requests:
+      cpu: 80m
+      memory: 128Mi
```

Comment:
these should probably be equal since this is a prod workload

Reply:
You mean limits = requests?